pkgs/by-name/ga/garnet/package.nix

```diff
@@ -5,6 +5,10 @@
   garnet,
   dotnetCorePackages,
 }:
+let
+  dotnet-sdk = with dotnetCorePackages; combinePackages [ sdk_6_0 sdk_7_0 sdk_8_0 ];
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
+in
```

Comment:
```suggestion
```
nit: i don't think thing we really need a `let ...; in` expression here
otherwise lgtm!

Reply:
yeah, that's fair.  I borrowed that from another package without thinking.  fixed.